src/core/zowe/core_for_zowe_sdk/sdk_api.py

```diff
@@ -21,7 +21,7 @@ class SdkApi:
     Abstract class used to represent the base SDK API.
     """
 
-    def __init__(self, profile, default_url):  
+    def __init__(self, profile, default_url, timeout=30):  
```

Comment:
Could this be generalized to `**session_args` so that all session arguments (including but not specific to timeout) can be overridden?
```python
    def __init__(self, profile, default_url, **session_args):
        ...
        self.session_arguments = {
            "verify": self.session.rejectUnauthorized,
            "timeout": 30,
            **session_args
        }
```